603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java

```diff
@@ -0,0 +1,55 @@
+package org.acme.spring.web;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.response.Response;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.*;
```

Comment:
remove `*` import

Reply:
there 5 occurrences ...

Reply:
Fixed. I used the Quarkus formatting: https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#idea-setup
Hope that's ok.